flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/connector/kafka/source/enumerator/KafkaSourceEnumerator.java

```diff
@@ -309,7 +309,8 @@ private void assignPendingPartitionSplits() {
 
     private AdminClient getKafkaAdminClient() {
         Properties adminClientProps = new Properties();
-        copyProperty(properties, adminClientProps, ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG);
+        adminClientProps.putAll(properties);
```

Comment:
It seems we also need to set the user properties in `getKafkaConsumer()` as well in this case.